dask/dataframe/multi.py

```diff
@@ -295,7 +295,34 @@ def merge_chunk(
                 args[0] = "inner"
             else:
                 kwargs["how"] = "inner"
-    out = lhs.merge(rhs, *args, **kwargs)
+    if len(args) and args[0] == "leftanti" or kwargs.get("how", None) == "leftanti":
```

Comment:
A few questions about this code block:

- This seems to be a lot of logic here. Perhaps this should be moved to an explicit `_leftanti_merge` helper function?
- Do we want to use the same logic for both cudf and pandas? Or, should we skip it for cudf like we do for leftsemi above?